s3backgrounddelete/s3backgrounddelete/object_recovery_msgbus.py

```diff
@@ -245,3 +245,23 @@ def purge(self):
             self._logger.error("Exception:{}".format(exception))
             self.__isproducersetupcomplete = False
             return False
+            
+    def get_count(self):
+        """Get count of unread messages."""
+        try:
+            consumer_group = self._config.get_msgbus_consumer_group()
+            if not self.__isproducersetupcomplete:
+                self.__setup_producer()
+                if not self.__isproducersetupcomplete:
+                    self._logger.info("producer connection issues")
+                    return False
+            if not self.__msgbuslib:
```

Comment:
__setup_producer() internally calls __loadmsgbuslibrary(). verify and remove this if block.

Reply:
done